third_party/2and3/yaml/__init__.pyi

```diff
@@ -22,10 +22,10 @@ def safe_load_all(stream: Union[str, IO[str]]) -> Iterator[Any]: ...
 def emit(events, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=...): ...
 def serialize_all(nodes, stream=..., Dumper=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...): ...
 def serialize(node, stream=..., Dumper=..., **kwds): ...
-def dump_all(documents, stream=..., Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...): ...
-def dump(data, stream=..., Dumper=..., **kwds): ...
-def safe_dump_all(documents, stream=..., **kwds): ...
-def safe_dump(data, stream=..., **kwds): ...
+def dump_all(documents: Sequence[Dict[str, Any]], stream=IO[str], Dumper=..., default_style=..., default_flow_style=..., canonical=..., indent=..., width=..., allow_unicode=..., line_break=..., encoding=..., explicit_start=..., explicit_end=..., version=..., tags=...) -> Any: ...
+def dump(data: Dict[str, Any], stream=IO[str], Dumper=..., **kwds) -> Any: ...
```

Comment:
It looks like documents can actually be any type: http://pyyaml.org/wiki/PyYAMLDocumentation.

Reply:
Done.